src/ya-comiste-backoffice/pages/products/create.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+
+import * as React from 'react';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import sx from '@adeira/sx';
+
+export default function ProductsCreatePage(): React.Node {
+  return (
+    <Formik
+      initialValues={{
+        name: '',
+        description: '',
+      }}
+      validate={(values) => {
+        const errors = {};
+        if (!values.name) {
+          errors.name = 'Product name is required';
+        }
+        if (!values.description) {
+          errors.description = 'Product description is required';
+        }
+        return errors;
```

Comment:
Formik plays really nice with [yup](https://github.com/jquense/yup), maybe not necessary if this is the only validation you will do though.

Reply:
I am aware of this one. I'd like to leave it for later since now I am just bootstrapping the pages. But yes, I will probably use it. :)